.vscode/settings.json

```diff
@@ -10,5 +10,6 @@
   "prettier.singleQuote": true,
   "[css]": {
     "editor.defaultFormatter": "vscode.css-language-features"
-  }
+  },
+  "conventionalCommits.scopes": ["components"]
```

Comment:
Hi @andreimuntean1, may I please ask if this setting should also be in this file?